bin/runtime/src/lib.rs

```diff
@@ -330,6 +330,7 @@ impl_opaque_keys! {
 
 parameter_types! {
     pub const SessionPeriod: u32 = DEFAULT_SESSION_PERIOD;
+    pub const MembersPerSession: u32 = 4;
```

Comment:
Added a constant to the primitives but also removed this type. Now we are using MembersPerSession from pallet_elections (this will allow to change nr of members in the future)